doc/nrf/index.rst

```diff
@@ -12,13 +12,13 @@ The |NCS| includes the Zephyr™ real-time operating system (RTOS), which is bui
 
    nRF53 Series devices are now supported for production.
    Wireless protocol stacks and libraries may indicate support for development or support for production for different series of devices based on verification and certification status.
-
-
-.. previous note about development and production support
-..   nRF53 Series devices (which are pre-production) and Zigbee and Bluetooth mesh protocols are supported for development in |release| for prototyping and evaluation.
-..   Support for production and deployment in end products is coming soon.
+   A "99" at the end of the version number of this documentation indicates continuous updates on the master branch since the previous major.minor release.
```

Comment:
I would add an empty line before this one, so that the note has two paragraphs. We're talking about two different things here.

Reply:
Fixed.